<ui version="4.0" >
 <class>RawDataSysSettingsCutsForm</class>
 <widget class="QWidget" name="RawDataSysSettingsCutsForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Direction</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>4</number>
      </property>
      <item>
       <widget class="VP1EtaPhiCutWidget" native="1" name="etaPhiCutWidget" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_cuts_idgeneral" >
     <property name="title" >
      <string>Cuts: Inner Detector General</string>
     </property>
     <layout class="QHBoxLayout" name="_2" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="margin" >
       <number>6</number>
      </property>
      <item>
       <layout class="QGridLayout" name="_3" >
        <property name="horizontalSpacing" >
         <number>-1</number>
        </property>
        <property name="verticalSpacing" >
         <number>0</number>
        </property>
        <item row="0" column="0" colspan="2" >
         <widget class="QLabel" name="label_9" >
          <property name="text" >
           <string>Show data from parts:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QCheckBox" name="checkBox_cut_indet_barrelA" >
          <property name="text" >
           <string>Barrel A</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QCheckBox" name="checkBox_cut_indet_barrelC" >
          <property name="text" >
           <string>Barrel C</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QCheckBox" name="checkBox_cut_indet_endcapA" >
          <property name="text" >
           <string>Endcap A</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QCheckBox" name="checkBox_cut_indet_endcapC" >
          <property name="text" >
           <string>Endcap C</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>1</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_cuts_trt" >
     <property name="title" >
      <string>Cuts: TRT</string>
     </property>
     <layout class="QHBoxLayout" name="_4" >
      <property name="margin" >
       <number>6</number>
      </property>
      <item>
       <layout class="QGridLayout" name="_5" >
        <property name="horizontalSpacing" >
         <number>-1</number>
        </property>
        <property name="verticalSpacing" >
         <number>0</number>
        </property>
        <item row="0" column="0" colspan="2" >
         <widget class="QCheckBox" name="checkbox_cut_trt_needHT" >
          <property name="toolTip" >
           <string>If set, do not show TRT PRD which have the 'isNoise' flag set.</string>
          </property>
          <property name="text" >
           <string>Require High Threshold</string>
          </property>
          <property name="checked" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QCheckBox" name="checkbox_cut_trt_TotMin" >
          <property name="toolTip" >
           <string>If set, do not show TRT PRD which have the 'isNoise' flag set.</string>
          </property>
          <property name="text" >
           <string>ToT min:</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QSpinBox" name="spinBox_cut_trt_ToTMin" >
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix" >
           <string> x 3.125ns</string>
          </property>
          <property name="maximum" >
           <number>24</number>
          </property>
          <property name="singleStep" >
           <number>1</number>
          </property>
          <property name="value" >
           <number>4</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QCheckBox" name="checkbox_cut_trt_TotMax" >
          <property name="toolTip" >
           <string>If set, do not show TRT PRD which have the 'isNoise' flag set.</string>
          </property>
          <property name="text" >
           <string>ToT max:</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QSpinBox" name="spinBox_cut_trt_ToTMax" >
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix" >
           <string> x 3.125ns</string>
          </property>
          <property name="maximum" >
           <number>24</number>
          </property>
          <property name="singleStep" >
           <number>1</number>
          </property>
          <property name="value" >
           <number>23</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>1</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Cuts: LUCID</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <property name="margin" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>QDC >=</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_lucid_qdc_min" >
        <property name="maximum" >
         <number>9999</number>
        </property>
        <property name="value" >
         <number>200</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>72</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VP1EtaPhiCutWidget</class>
   <extends>QWidget</extends>
   <header>VP1Base/VP1EtaPhiCutWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
